bin/cmsl1t_dirty_batch

```diff
@@ -214,6 +272,7 @@ def run(config_file, debug, batch, files_per_job):
 
```

Comment:
Perhaps we should add a warning message to this command that in the near future this will be removed completely and that people should now use the `cmsl1t_batch` command instead?

Reply:
That's a good point, will add that